src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java

```diff
@@ -738,6 +738,11 @@ public ArticlePageObject openRandomArticle(String wikiURL) {
     getUrl(wikiURL + URLsContent.SPECIAL_RANDOM);
     return new ArticlePageObject(driver);
   }
+  
+  public ArticlePageObject openMainPage(String wikiURL) {
```

Comment:
Actually there are differences between article page and main page
was created with intend to switch to MainpagePageObject if such entity becomes created
hopefully it will within this PR